source/index.js

```diff
@@ -105,139 +93,87 @@ const np = async (input = 'patch', options, {pkg, rootDir, isYarnBerry}) => {
 
 	const shouldEnable2FA = options['2fa'] && options.availability.isAvailable && !options.availability.isUnknown && !pkg.private && !npm.isExternalRegistry(pkg);
 
-	// Yarn berry doesn't support git commiting/tagging, so use npm
-	const shouldUseYarnForVersioning = options.yarn === true && !isYarnBerry;
-	const shouldUseNpmForVersioning = options.yarn === false || isYarnBerry;
-
 	// To prevent the process from hanging due to watch mode (e.g. when running `vitest`)
 	const ciEnvOptions = {env: {CI: 'true'}};
 
 	const tasks = new Listr([
 		{
 			title: 'Prerequisite check',
 			enabled: () => options.runPublish,
```

Comment:
I haven't tested it, but can these just be booleans?

```suggestion
			enabled: options.runPublish,
```

Reply:
I don't know but I don't want to add to the scope of this so I'd rather not touch.